components/eventing-controller/pkg/handlers/beb.go

```diff
@@ -284,6 +287,14 @@ func (b *BEB) getSubscription(name string) (*types.Subscription, error) {
 	return bebSubscription, nil
 }
 
+func getCleanEventTypes(events types.Events) []string {
+	cleanEventTypes := make([]string, 0, len(events))
+	for _, e := range events {
+		cleanEventTypes = append(cleanEventTypes, e.Type)
```

Comment:
how about:
```
for i, e := range events {
		cleanEventTypes[i] = e.Type
```

Reply:
Agree, not smart to create with size and then don't address by index.